src/videojs-http-streaming.js

```diff
@@ -641,11 +642,11 @@ class VhsHandler extends Component {
       'initialPlaylistSelector',
       'experimentalBufferBasedABR',
```

Comment:
Also, we do want to rename this to `bufferBasedABR` - but leave it defaulting to `false` for the time being.